src/routes/Landing.tsx

```diff
@@ -0,0 +1,338 @@
+import { Mobile } from "styled-icons/icomoon/Mobile";
+import { LocationOff } from "styled-icons/material/LocationOff";
+import { Cached } from "styled-icons/material/Cached";
+import { LogInCircle } from "styled-icons/boxicons-regular/LogInCircle";
+
+const HeartIcon = styled(Heart)`
+  width: 10px;
+  height: 10px;
+  margin: 0 6px;
+`;
+
+interface Props extends RouteComponentProps {}
+
+const LandingGlobalStyle = createGlobalStyle`
+  body {
+    background: ${themeGet("colors.white")} !important;
+  }
+`;
+
+const Section = styled(Box).attrs({ as: "section" })``;
+
+const underlineAnimation = keyframes`
+  from { width: 0; }
+  to { width: 100%; }
+`;
+
+const Underline = styled(Box)`
+  position: absolute;
+  bottom: -8px;
+  left: 0;
+  width: 0;
+  height: 8px;
+  background: url(${underline});
+  background-repeat: repeat-x;
+  background-position: center bottom;
+  background-size: 200px 8px;
+  animation: ${underlineAnimation} 0.3s 3s ease-in-out forwards;
+`;
+
+const Screenshot = styled(Box)`
+  position: absolute;
+  left: 80ch;
+  bottom: -20px;
+  width: 1275px
+  height: 670px;
+  background: url(${retrotoolScreenshot}) no-repeat;
+  filter: drop-shadow(0 0 1px rgba(0,0,0,.5));
+`;
+
+const RetroSteps = styled(Text).attrs({
+  as: "h3",
+  textStyle: "landingHowtoStep"
+})`
+  ::before {
+    content: "";
+    position: absolute;
+    margin-top: 12px;
+    margin-left: -40px;
+    width: 30px;
+    height: 1px;
+    background: ${themeGet("colors.violet")};
+  }
+`;
+
+const Landing: React.FC<Props> = () => (
+  <>
+    <LandingGlobalStyle />
+    <Box bg="white" margin="auto">
+      <Flex
+        as="header"
+        justifyContent="space-between"
+        alignItems="center"
+        p={6}
+      >
+        <Logo />
+        <Flex>
+          <Text mr={3}>
+            <a href="https://twitter.com/marciobarrios" tabIndex={3}>
+              Contact us
+            </a>
+          </Text>
+          ·
+          <Text ml={3}>
+            <a href="https://github.com/retro-tool/" tabIndex={4}>
+              Github
+            </a>
+          </Text>
+        </Flex>
+      </Flex>
+      <Flex as="main" flexDirection="column">
+        <Section
+          textAlign={["left", null, "center"]}
+          maxWidth="100ch"
+          margin="auto"
+          py={[24, null, null, 120, 150]}
+          px={6}
+        >
+          <Text as="h1" fontSize={[7, 8, 9, 10]} lineHeight={7}>
+            Retro tool,{" "}
+            <Box position="relative" as="span">
+              simple and effective{" "}
+              <Underline display={["none", null, null, "block"]} />
+            </Box>{" "}
+            retrospectives for your team
+          </Text>
+          <Box mt={6} mb={[8, null, 10]} mx="auto" maxWidth="60ch">
+            <Text fontSize={[5, null, 6]} color="grey">
+              Collaborate with your remote team to improve what you do, guided
+              by a simple workflow. And for free!
```

Comment:
I would remove the `For free` part, as that's already in the notes below.

Reply:
I would highlighted here that is quite prominent.

Reply:
I'd prefer not to, but let's see what others think

Reply:
I believe this is key and should be said _above the fold_. I think it does a good job communicating the three essential things: **simple**, **no-bullshit** and **free**.

I wouldn't touch this.